<ui version="4.0" >
 <class>OTRPrefsUI</class>
 <widget class="QWidget" name="OTRPrefsUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>459</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="twSettings" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="settings" >
      <attribute name="title" >
       <string>&amp;Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox1" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Private Keys</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="2" >
           <widget class="QPushButton" name="btGenFingerprint" >
            <property name="text" >
             <string>&amp;Generate</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cbKeys" >
            <property name="maximumSize" >
             <size>
              <width>400</width>
              <height>32767</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel2" >
            <property name="text" >
             <string>Account:</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3" >
           <widget class="QLabel" name="tlFingerprint" >
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text" >
             <string>No Fingerprint</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QGroupBox" name="gbPolicy" >
         <property name="title" >
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="kcfg_rbAlways" >
            <property name="toolTip" >
             <string>Always encrypt messages</string>
            </property>
            <property name="text" >
             <string>Al&amp;ways</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QRadioButton" name="kcfg_rbManual" >
            <property name="toolTip" >
             <string>Encrypt messages if the other side requests an OTR connection</string>
            </property>
            <property name="text" >
             <string>&amp;Manual</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="kcfg_rbOpportunistic" >
            <property name="toolTip" >
             <string>Encrypt messages automatically if the other side supports OTR</string>
            </property>
            <property name="text" >
             <string>&amp;Opportunistic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QRadioButton" name="kcfg_rbNever" >
            <property name="toolTip" >
             <string>Never encrypt messages</string>
            </property>
            <property name="text" >
             <string>Ne&amp;ver</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fingerprints" >
      <attribute name="title" >
       <string>K&amp;nown Fingerprints</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QTableWidget" name="tbFingerprints" >
         <property name="columnCount" >
          <number>5</number>
         </property>
         <column>
          <property name="text" >
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>bla</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>New Column</string>
          </property>
         </column>
         <column/>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="btVerify" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>&amp;Verify Fingerprint</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="btForget" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>F&amp;orget Fingerprint</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
